src/Maestro/Microsoft.DotNet.Maestro.Tasks/Microsoft.DotNet.Maestro.Tasks.csproj

```diff
@@ -24,5 +24,11 @@
     <ProjectReference Include="..\Client\src\Microsoft.DotNet.Maestro.Client.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
We need to add this snippet into this project so that the symbols are available for code coverage: 

```
  <PropertyGroup>
    <DebugType>Full</DebugType>
  </PropertyGroup>
```

For reference: https://github.com/Microsoft/vstest/issues/800

Reply:
Does that flag have an effect on the release configuration code? Should it be bumpered to check for debug or release configuration?

Reply:
Not that I'm aware of. All of our other .NET Core projects are configured the same way and this property is called out in the Validation documentation, too